bundles/binding/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/HueGenericBindingProvider.java

```diff
@@ -84,10 +84,12 @@ public void processBindingConfiguration(String context, Item item,
 
 		super.processBindingConfiguration(context, item, bindingConfig);
 
-		try {
-
-			if (bindingConfig != null) {
+    if( bindingConfig == null || bindingConfig.length() < 1 ) {
+        logger.warn("No settings configured for Hue binding for item '"+item+"'.");
+        return;
+    }
```

Comment:
Well, I don't use any IDE. (Luddite here!)

Hopefully the reformat I just did was sufficient?